launch_ros/launch_ros/actions/load_composable_nodes.py

```diff
@@ -161,10 +159,17 @@ def execute(
         context: LaunchContext
     ) -> Optional[List[Action]]:
         """Execute the action."""
+        # resolve target container node name
+        if isinstance(self.__target_container, ComposableNodeContainer):
```

Comment:
Will this work if someone were to subclass `ComposableNodeContainer`? Ideally it would. I don't remember what the gotcha's are there, but I know we have this utility for some reason: https://github.com/ros2/launch/blob/a46aabdde1f58c7e2e5c056c0c4a1ea830faac5c/launch/launch/utilities/class_tools_impl.py#L34 which uses `issubclass()` internally.

Reply:
@wjwwood so at least according to [this](https://stackoverflow.com/a/1549854) it seems like it should, but I'm happy to use the created util function instead

Reply:
As mentioned above, ended but using `is_a_subclass`

Reply:
@wjwwood I don't understand `is_a_subclass` implementation.
[isinstance](https://docs.python.org/2/library/functions.html#isinstance) already returns `True` if the object is instance of a subclass.
[issubclass](https://docs.python.org/2/library/functions.html#issubclass) returns `True` if a class is a subclass of another (and not when an object is instance of a subclass).
So I don't understand why `is_a_subclass` checks for both `isinstance` and `issubclass`.

IMO, using [is_a](https://github.com/ros2/launch/blob/a46aabdde1f58c7e2e5c056c0c4a1ea830faac5c/launch/launch/utilities/class_tools_impl.py#L25) util in launch or `isinstance` is ok.

Reply:
@ivanpauno Yeah, I don't remember a rationale off-hand, so maybe your recommendation is sufficient.